addons/website/controllers/main.py

```diff
@@ -773,7 +773,16 @@ def get_seo_data(self, res_id, res_model):
             res['can_edit_seo'] = False
         record = record.sudo()
 
-        res.update(record.read(fields)[0])
```

Comment:
Seems introduced by https://github.com/odoo/odoo/commit/45ea6e4a4a5c8e314b110a45198fbe3d57bb996e

Can we add a reference to that commit inside your commit message?

I'm wondering if it's useful at all to do that call if the model doesn't inherit from "website.seo.metadata".
Maybe we could do a pre-condition at the top of this controller method, something like:

```py
if 'website_meta_title' not in request.env[res_model]._fields:
    # model does not inherit from 'website.seo.metadata' mixin
    return {}
```
(Ideally it would be done at JS level to prevent the RPC but I'm not sure we have that info there).

The "checkAndNotifySEO" seems like it will do nothing if it receives an empty result, I have not tested it myself though.

cc @xO-Tx @qsm-odoo

Reply:
Indeed, looks like it's doing close to the same check:
`'data-can-optimize-seo': 'website_meta_description' in main_object.fields_get(),`

We can modify it in this PR as it already exists, and then you give us your review blessing :slightly_smiling_face:

Reply:
Hello @awa-odoo, @xO-Tx 

made the changes as requested,
please have a look if there is anything left!

Thanks